### ISSUE TITLE
match "color:" slack webhook value to Mantis status colour

### ISSUE THREAD

**Issue by RedSpid3r** [infojunkie/MantisBT-Slack#25]
Is it possible to have the "color":"#" option in the Slack webhook be dynamic and reflect the colours from the config file so that if a status is changed the corresponding slack post matches

**Comment by infojunkie** [comment]
Good idea! Will implement time permitting, and feel free to submit a PR as well.

**Comment by AmauryBs** [comment]
Hello, I've matched the sclack color with status color. I'm not sure that I used the most efficient way to do so but you can use it to improve the plugin if you want

**Comment by infojunkie** [comment]
Thanks! Can you point to your code for review? Also, I'm out on vacation for 3 weeks so I'll be slow to respond.

**Comment by AmauryBs** [comment]
Here is my code: https://github.com/AmauryBs/MantisBT-Slack (I also made a pull request). I've changed mainly changed the get channel function. I also added a check list to activate or not those fonctionnalities and a print of the actual configuration in the configuration page. I wish you nice vacations =)
